analysis/ActivityTools.py

```diff
@@ -225,13 +221,17 @@ def DetectedCoincidences( DecayRates, DecayData, SimulationWindow, CoincidenceWi
 
         # Classify the events
         if TwoHitEvent( event, DetectorRadius, ZMin, ZMax ):
-            hitRadii.append( FindHitRadius( event, DetectorRadius ) )
+            hitRadius, hasSameEventID = FindHitRadius( event, DetectorRadius )
```

Comment:
Since the hit radius and the eventID are two different concepts, why are they both returned from  this one method? Better to make a specific method for the additional eventID test.

Reply:
Added a dedicated function